webapi/appsettings.json

```diff
@@ -223,7 +220,10 @@
   // Filter for hostnames app can bind to
   "AllowedHosts": "*",
   // CORS
-  "AllowedOrigins": ["http://localhost:3000", "https://localhost:3000"],
+  "AllowedOrigins": [
+    "http://localhost:3000",
```

Comment:
Do we need to enable http?

Reply:
This is just a whitespace change (autoformatting from VS Code), no functional changes.